subsys/usb/class/dfu/usb_dfu.c

```diff
@@ -53,6 +53,10 @@
 #include <usb_work_q.h>
 
 #define LOG_LEVEL CONFIG_USB_DEVICE_LOG_LEVEL
+#ifdef CONFIG_USB_DFU_REBOOT
```

Comment:
No need to include `#include <sys/reboot.h>` conditional.

Reply:
Fixed.